pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/constants/Constants.java

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.ddp.constants;
+
+public class Constants {
```

Comment:
File PdfConstants didnt seem to be in use and the constants are used in both pdf generation and elastic data extraction, hence renamed to Constants.java so that t looks common for DSS constants, however if Constants.java under package constants is confusing, maybe we can use some other name to reflect common DSS constants like DSSConstants.java or CommonConstants.java or CoreConstants.java or something else.